x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -117,27 +117,21 @@ export const ProcessEventDot = styled(
 
       const labelYHeight = markerSize / 1.7647;
 
-      const levelAttribute = adjacentNodeMap?.level
-        ? {
-            'aria-level': adjacentNodeMap.level,
-          }
-        : {};
-
-      const flowToAttribute = adjacentNodeMap?.nextSibling
-        ? {
-            'aria-flowto': adjacentNodeMap.nextSibling,
-          }
-        : {};
+      /**
+       * An element that should be animated when the node is clicked.
+       */
+      const animationTarget: { current: SVGAnimationElement | null } = React.createRef();
```

Comment:
Changed variable to `animationTarget`. It is an element that is animated when the node is clicked, but not always the target of the click event.